src/containers/Tenant/utils/schemaQueryTemplates.ts

```diff
@@ -54,66 +60,79 @@ PARTITION BY HASH(id)
 WITH (STORE = COLUMN)`;
 };
 export const createAsyncReplicationTemplate = () => {
-    return `CREATE OBJECT secret_name (TYPE SECRET) WITH value="secret_value";
+    return `-- docs: https://ydb.tech/docs/en/yql/reference/syntax/create-async-replication
+CREATE OBJECT secret_name (TYPE SECRET) WITH value="secret_value";
 
 CREATE ASYNC REPLICATION my_replication
-FOR \`/remote_database/table_name\` AS \`local_table_name\` --[, \`/remote_database/another_table_name\` AS \`another_local_table_name\` ...]
+FOR \${1:<original_table>} AS \${2:replica_table} --[, \`/remote_database/another_table_name\` AS \`another_local_table_name\` ...]
 WITH (
-    CONNECTION_STRING="grpcs://mydb.ydb.tech:2135/?database=/remote_database",
+    CONNECTION_STRING="grpcs://mydb.ydb.tech:2135/?database=/\${3:<remote_database>}",
```

Comment:
It's not enough to change database here, the whole connection string should be updated. For our dev slice it will be something like `grpc://ydb-vla-dev02-000.search.yandex.net:31001/?database=/dev02/home/xenoxeno/db1` 

Maybe the whole connection string should be a snippet?

And as for the comments, I don't think we need to update values there. Otherwise all comments fields should be snippets - endpoint, database, user, password

Reply:
Agree with you!